fe/fe-core/src/main/java/org/apache/doris/planner/BrokerScanNode.java

```diff
@@ -454,6 +476,10 @@ private void processFileGroup(
                 } else {
                     TBrokerRangeDesc rangeDesc = createBrokerRangeDesc(curFileOffset, fileStatus, formatType,
                             leftBytes, columnsFromPath, numberOfColumnsFromFile, brokerDesc);
+                    if (this instanceof HiveScanNode) {
```

Comment:
It better to implement a processFileGroup in HiveScanNode and fill in the HDFSParams